### PR TITLE
replace "google.common.base.Preconditions" API with jdk API

### PR DESCRIPTION
**Pull request by zzq1314zll**
replace "google.common.base.Preconditions" API with jdk API,
for #6498

### CODE REVIEW

File: common/src/main/java/com/alibaba/nacos/common/utils/Preconditions.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.common.utils;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import static com.google.common.base.Strings.lenientFormat;
```
Comment (KomachiSion) [inline common/src/main/java/com/alibaba/nacos/common/utils/Preconditions.java:21]: still depend guava?

**Review by KomachiSion — CHANGES_REQUESTED** [review]
It seems that full copy from guava's Preconditions?

File: common/src/main/java/com/alibaba/nacos/common/utils/Preconditions.java (hunk 1 above)
Comment (KomachiSion) [reply common/src/main/java/com/alibaba/nacos/common/utils/Preconditions.java:21]: I think the Nullable is no need, we can judge in method.

And the guava depend can be refactor.

**Comment by zzq1314zll** [comment]
No, 

> It seems that full copy from guava's Preconditions?

NO! 我仅仅是借鉴，没有有copy

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**